ramls/schema/circulation-rule-matches.json

```diff
@@ -1,19 +1,19 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Loan policy ids schema",
+  "title": "Circulation policy ids schema",
   "type": "object",
   "properties": {
-    "loanRuleMatches": {
-      "id": "loanRuleMatches",
+    "circulationRuleMatches": {
```

Comment:
similarly, this could be `ruleMatches`?

Reply:
Changed in https://github.com/folio-org/mod-circulation/commit/4b0a55b49ecc627393e69877032f53a9554e7d7f